.eslintrc

```diff
@@ -0,0 +1,26 @@
+parserOptions:
+  ecmaVersion: 7
+  sourceType: module
+extends:
+  - eslint:recommended
+  - prettier
```

Comment:
yeah `eslint-prettier-config` basically turns off every rule that might potentially conflict with `prettier`, including `max-len`.